docs/Windows.md

```diff
@@ -61,7 +61,19 @@ You will need the ICU libraries from [ICU - International Components for Unicode
 
 #### Building Swift code
 
-You should use a CMake project to build a Swift program. An example CMake project with support for mixed-language support is available at [https://github.com/compnerd/swift-cmake-demo](https://github.com/compnerd/swift-cmake-demo), sample commands that build the project are documented there. The make commands are to be executed from the Native Tools Command Prompt. Note that the cmake executable to be used is the one mentioned above. (Try `cmake -version` to see which CMake version is actually being called.)
+You should use a CMake project to build a Swift program. As an example CMake project use the "HelloWorld-CMake" example from [https://github.com/compnerd/swift-cmake-demo](https://github.com/compnerd/swift-cmake-demo) and use the following commands from within the project directory to build the project for Windows:
+
+```cmd
```

Comment:
The library is part of the install, so perhaps the installation_dir should be C:?

Reply:
Done.